### ISSUE TITLE
Please make the g4 and tokens files readable for a dark laf

### ISSUE THREAD

**Issue by Chris2011** [fredyvonvinet/ANTLR4-Plugins-for-NetBeans#38]
I use the darcula look and feel of NetBeans, and for this, the grammar and tokens files doesn't look readable. Would be great to see a fix soon.

![grammar-not-readable](https://user-images.githubusercontent.com/795658/28277926-a605c574-6b1b-11e7-9491-32566be44ba3.png)

Regards

Chris

**Comment by fredyvonvinet** [comment]
I understand your need but it is not possible to satisfy all users that have different color themes for text rendering. So I decoded to define one adapted for standard NetBeans theme.

But I have the solution to your problem. It's you! No, it is not a bad joke! ANTLR Color theme is just completely integrated in NetBeans text rendering. So you may change colors. You can find them in Tools/Options/Fonts & Colors. Select ANTLR v4 grammar in language combo box. As you can see, you can modify the way tokens file are rendered as well.

**Comment by Chris2011** [comment]
Yes I know that, but for this needs I created a ticket at the darcula repo to ask who is responsible for the color problems of 3rd-party-plugins: https://github.com/Revivius/nb-darcula/issues/116 and he referenced a NetBeans bug: https://netbeans.org/bugzilla/show_bug.cgi?id=257867

It is about Code completion, but I don't know whether this will help for normal colors too. It is about to use LFCustoms.shiftColor(). Could this help?
